src/main/java/uk/ac/ebi/ena/webin/cli/context/genome/GenomeManifestReader.java

```diff
@@ -192,12 +190,6 @@ public GenomeManifestReader(WebinCliParameters parameters, MetadataProcessorFact
             .and()
             .file()
             .optional()
-            .name(Field.AGP)
-            .desc(Description.AGP)
-            .processor(getAgpProcessors())
-            .and()
-            .file()
-            .optional()
```

Comment:
if you intend to remove AGP file information from field definition then include .file().optional() lines above and leave .and().file().optional() lines at the bottom.

field definition starts with the type of the field followed by whether its optional or required and then other information. the method and() marks the end of previous field definition and start of a new one.

Reply:
yes below are the lines I intend to remove: 
`.and()
.file()
.optional()
.name(Field.AGP)
.desc(Description.AGP)
.processor(getAgpProcessors())`

the changed lines display is not great, it just keeps the and, file, and optional (lines 192, 193, and 194 and removes the ones from below, 198, 199, and 200. The result is still the same and the above and below descriptors of flatfile and chromosome list are intact. Please let me know if I am not understanding correctly. Thanks.

Reply:
oh ok. i see it now. thanks for clarifying.